subsys/bluetooth/host/hci_core.c

```diff
@@ -423,14 +427,16 @@ int bt_hci_cmd_send_sync(uint16_t opcode, struct net_buf *buf,
 		}
 	}
 
-	LOG_DBG("rsp %p opcode 0x%04x len %u", buf, opcode, buf->len);
+	LOG_DBG("rsp %p opcode 0x%04x len %u", cmd(buf)->rsp, opcode, cmd(buf)->rsp->len);
 
 	if (rsp) {
-		*rsp = buf;
+		*rsp = cmd(buf)->rsp;
 	} else {
-		net_buf_unref(buf);
+		net_buf_unref(cmd(buf)->rsp);
```

Comment:
Is `cmd(buf)->rsp` always non-NULL here?

Reply:
Changed :)